DOCKER.md

```
@@ -0,0 +1,307 @@
+
+### Option A: Using docker-compose (if available)
+
+### 1. Ensure `.env` file exists
+
+```bash
+cp .env.example .env
+# Edit .env with your bot credentials
+```
+
+Your `.env` should contain:
+```env
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)